tests/ContentRepository90/Rules/FusionNodeTypeNameRector/Fixture/some_file.fusion.inc

```diff
@@ -0,0 +1,36 @@
+prototype(Neos.Fusion.Form:Checkbox)  < prototype(Neos.Fusion.Form:Component.Field) {
+
+renderer = Neos.Fusion:Component {
+
+#
+# pass down props
+#
+attributes = ${node.nodeType.name || documentNode.nodeType.name}
+renderer = afx`
+<input
+        name={node.nodeType.name}
+        value={someOtherVariable.nodeType.name}
+        {...node.nodeType.name}
+/>
+`
+}
+}
+-----
+// TODO 9.0 migration: Line 13: You may need to rewrite "VARIABLE.nodeType.name" to VARIABLE.nodeTypeName.value. We did not auto-apply this migration because we cannot be sure whether the variable is a Node.
```

Comment:
just wondering: Is there a specific reason why we don't add this comment above the affected line? Would that be more complicated?

Reply:
That's how all Fusion migrations work. AFAIS it's not that easy to change.

Reply:
OK, great. I didn't really look at the other ones yet tbh 😊